frontend/src/components/data-table/chart-transforms/storage.ts

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2024 Marimo. All rights reserved. */
+
+import type { CellId } from "@/core/cells/ids";
+import type { TypedString } from "@/utils/typed";
+import { atomWithStorage } from "jotai/utils";
+import type { z } from "zod";
+import { atom } from "jotai";
+import type { ChartSchema } from "./chart-schemas";
+
+export type TabName = TypedString<"TabName">;
+export const KEY = "marimo:charts";
```

Comment:
nit: could do `marimo:charts:v1` so easier to identify how to bust the localStorage for the next person who might touch this code